packages/cpp/ArmoniK.Api.Tests/source/SubmitterClientTest.cpp

```diff
@@ -253,26 +256,31 @@ TEST(testMock, getResult) {
   TaskOptions task_options;
   armonik::api::grpc::v1::ResultRequest result_request;
 
-  init(channel, task_options);
+  init(channel, task_options, log);
 
   auto stub = armonik::api::grpc::v1::results::Results::NewStub(channel);
 
   grpc::ClientContext context;
 
+  log.info("Creating Client");
```

Comment:
I think this kind of log should be debug level

Reply:
done